services/fuzzing-decision/src/fuzzing_decision/decision/pool.py

```diff
@@ -161,12 +163,19 @@ def cancel_tasks(worker_type):
     hooks = taskcluster.get_service("hooks")
     queue = taskcluster.get_service("queue")
 
+    newer_than = datetime.utcnow().replace(tzinfo=timezone.utc) - timedelta(
```

Comment:
I'd call this `cancel_limit` or something. Makes things a bit more readable when it's used.